CoreFoundation/String.subproj/CFString.c

```diff
@@ -5713,8 +5711,8 @@ void CFStringNormalize(CFMutableStringRef string, CFStringNormalizationForm theF
                         if (CFUniCharIsSurrogateHighCharacter(nextChar) && ((contents + 1) < limit) && CFUniCharIsSurrogateLowCharacter(*(contents + 1))) {
                             nextChar = CFUniCharGetLongCharacterForSurrogatePair(nextChar, *(contents + 1));
                             if (!CFUniCharIsMemberOfBitmap(nextChar, (const uint8_t *)CFUniCharGetBitmapPtrForPlane(kCFUniCharNonBaseCharacterSet, (nextChar >> 16))) || (0 != CFUniCharGetCombiningPropertyForCharacter(nextChar, (const uint8_t *)CFUniCharGetUnicodePropertyDataForPlane(kCFUniCharCombiningProperty, (nextChar >> 16))))) break;
-                        } else {
-                            if (!CFUniCharIsMemberOfBitmap(nextChar, nonBaseBMP) || (0 != CFUniCharGetCombiningPropertyForCharacter(nextChar, combiningBMP))) break;
```

Comment:
The logic was more clear in the original version; the `if` statement here parallels that above. Please revert to the original.